src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
@brett in a world where r_t is testing remote nodes, we are going to need to copy them to a local directory in order to upload them to GiddyUp.  Therefore, we should always copy the logs to the results directory.

I think it sufficient to notify @ooshlablu and @gcymbalski that r_t is storing the logs with the results locally, and they script out a reaping policy.  We should also add a note to the changes file to let users know that their local results directory may grow as well.